Formula/grcov.rb

```diff
@@ -0,0 +1,32 @@
+class Grcov < Formula
+  desc "Rust tool to collect and aggregate code coverage data for multiple source files"
+  homepage "https://github.com/mozilla/grcov"
+  url "https://github.com/mozilla/grcov/archive/v0.7.1.tar.gz"
+  sha256 "cb7167ee8770d2903839e00d4a631cca08fb8095b2339c8796e7b9510d5f5055"
+  license "MPL-2.0"
+  head "https://github.com/mozilla/grcov.git"
+
+  depends_on "rust" => :build
```

Comment:
```suggestion
  depends_on "rust" => [:build, :test]
```
You need this to use `cargo` in the `test` block.

Reply:
Ooh, thanks